manifests/mod/passenger.pp

```diff
@@ -56,13 +56,14 @@
 
   $_id = $mod_id
   $_path = $mod_path
-  ::apache::mod { 'zpassenger':
+  ::apache::mod { 'passenger':
     package        => $_package,
     package_ensure => $_package_ensure,
     lib            => $_lib,
     lib_path       => $_lib_path,
     id             => $_id,
     path           => $_path,
+    loadfile_name  => 'zpassenger.load',
```

Comment:
cool, I fully agree.
In my defense, that parameter didn't exist when I first wrote that patch :)